### PR TITLE
Add debuginfo flags for libabigail

### PR DESCRIPTION
**Pull request by hainest**

### CODE REVIEW

**Comment by vsoch** [comment]
@hainest where are these set?

File: spliced/predict/libabigail.py [hunk 1]
```diff
@@ -139,7 +139,15 @@ def run_abidiff(self, original_lib, replace_lib):
         """
         Run abi diff with an original and comparison library
         """
-        command = "%s %s %s" % (self.abidiff, original_lib, replace_lib)
+        debug1=""
+        if os.environ['LIBABIGAIL_DEBUGINFO_DIR1']:
```
Comment (vsoch) [inline spliced/predict/libabigail.py]: os.environ will only have a key if it's defined, so this is going to error if not. You should do:

```python
debug1 = os.environ.get("LIBABIGAIL_DEBUGINFO_DIR1", "")
if debug1:
    debug1 = f"--debug-info-dir1={debug1}"
```
A few questions / notes about the above:

- should we instead have key=val (as I've done?)
- are there really FOUR slashes for the arg?

Comment (vsoch) [reply spliced/predict/libabigail.py]: Also per the ABI lab script you set, don't we need to add that too (this is just for libabigail).

Comment (hainest) [reply spliced/predict/libabigail.py]: Ah, I didn't know that about os.environ. We should use your method, instead.

>are there really FOUR slashes for the arg?

Which arg?

Comment (vsoch) [reply spliced/predict/libabigail.py]: And there should be a note in the docs here to set these envars (and an example to do it) and description for what they are. Ty!

Comment (vsoch) [reply spliced/predict/libabigail.py]: For both you put

```
----debug-info-dir2
```
and I'd typically expect
```
--debug-info-dir2
```

Comment (hainest) [reply spliced/predict/libabigail.py]: >os.environ will only have a key if it's defined, so this is going to error if not

Fixed to use environ.get

>should we instead have key=val (as I've done?)

It depends on how libabigail is parsing command line arguments. It _should_ accept both forms, but I've only ever used the form with spaces.

>are there really FOUR slashes for the arg?

Nope. That was a big copy-pasta error. Derp.

>Also per the ABI lab script you set, don't we need to add that too (this is just for libabigail).

The usages are different. Here, we are executing abidiff directly, so we need to construct and pass the arguments. For abi lab, we are using python to execute a shell script that then executes abi-dumper. That extra level of indirection allows for passing the environment variables without any interaction from us.

>And there should be a note in the docs here to set these envars (and an example to do it) and description for what they are.

Added, but should be double-checked for readability and docstring conventions.

**Review by vsoch — APPROVED** [review]

**Pull request merged**